frontend/src/app/core/services/issue.service.ts

```diff
@@ -24,10 +26,20 @@ export class IssueService {
         return this.httpService.getRequest<IssueInfo[]>(`${this.routePrefix}/info/${memberId}`);
     }
 
-    public getIssuesInfoLazy(memberId: number, event: LazyLoadEvent, status?: IssueStatus):
+    public getIssuesInfoLazy(memberId: number, event: LazyLoadEvent, status?: IssueStatus, project?: Project):
     Observable<{ collection: IssueTableItem[], totalRecords: number }> {
+        let params = new HttpParams();
+
+        if (status !== undefined) {
```

Comment:
Could you use just if(status) ?

Reply:
I think no, because status is enum of 0-2 numbers. 
And if you write `if(status)` it will accept `number 0` and `undefined`. But in this case we need only check for undefined.

Reply:
ok, but it is Bohdan' PR so he should give an answer